docs/SPEC.md

```diff
@@ -2757,6 +2829,56 @@ If you need to convert other columns use the `map` function directly with the `u
 
 [IMPL#242](https://github.com/influxdata/platform/issues/242) Update specification around type conversion functions.
 
+#### Errors
+
+Errors selects error tables from the meta channel and places them on the data channel.
+The input data channel is dropped.
+The output meta channel is empty.
+
+Example:
+
+    // Perform a simple downsample
+    from(bucket:"telegraf/autogen")
+        |> range(start: -5m)
+        |> aggregateWindow(every: 1m, fn: mean)
+        |> to(bucket: "telegraf/5m")
+        |> errors()
+        // Send all errors during downsampling to S3
+        |> s3.to(bucket: "example.com/errors")
+
+
+#### Meta
+
+Meta places all tabels from the meta channel into the data channel.
+The input data channel is dropped.
+The output meta channel is empty.
+
+Example:
+
+    // The errors function is implemented in terms of the meta function.
+    errors = (tables=<-) => 
+        tables
+            |> meta()
+            |> filter(fn: (r) => r.name == "errors")
+
+
+#### Stats
+
+Stats selects stats tables from the meta channel and places them on the data channel.
+The input data channel is dropped.
+The output meta channel is empty.
+
+Example:
+
+    // Ship data to S3
+    from(bucket:"telegraf/autogen")
+        |> range(start: -5m)
+        |> s3.to(bucket: "example.com/telegraf")
+        |> stats()
```

Comment:
same as for errors, can we have some sort of filtering here?   compared to `meta()`  the point is to filter meta-data conveniently.  let's take it a step further.